Sources/NIOHTTP1/HTTPDecoder.swift

```diff
@@ -186,14 +185,29 @@ public class HTTPDecoder<HTTPMessageT>: ByteToMessageDecoder, AnyHTTPDecoder {
     private var parser = http_parser()
     private var settings = http_parser_settings()
 
-    fileprivate var pendingInOut: [NIOAny] = []
     fileprivate var state = HTTPParserState()
 
     fileprivate init(type: HTTPMessageT.Type) {
         /* this is a private init, the public versions only allow HTTPClientResponsePart and HTTPServerRequestPart */
         assert(HTTPMessageT.self == HTTPClientResponsePart.self || HTTPMessageT.self == HTTPServerRequestPart.self)
     }
 
+    deinit {
```

Comment:
I used a `deinit { }` for it as `decoderRemoved` may be called while still in the `decode(...)` and so I can not modify the `parser`.